styles/styles.css

```diff
@@ -78,7 +113,7 @@ blockquote {
 
 code,
 pre {
-  font-size: var(--body-font-size-s);
+  font-size: var(--spectrum-body-size-s);
```

Comment:
do code snippets exist in current blog posts?

Reply:
a good place to check would be the adobe design repo

Reply:
The old/current Adobe Design repo isn't styling <code> or <pre> elements at all! I'll remove these styles.